src/bindgen/parser.rs

```diff
@@ -659,9 +659,10 @@ impl Parse {
                         loggable_item_name()
                     );
                 }
-                (false, _) => {
+                (false, Some(_exported_name)) => {
```

Comment:
I would just use `(_)` or `(..)`, fwiw. But this works for me as well

Reply:
I can change it to `_` next time :)